.gitignore

```diff
@@ -13,4 +13,6 @@ deployments/localhost/**
 coverage/
 coverage.json
 
-*.txt
\ No newline at end of file
+*.txt
+.env
```

Comment:
.env already exists at the top.

Maybe separate this by

#VSCode
workspace.code-workspace